rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
```

Comment:
- [ ] When I try to create a rental, I get this error in the console

![image](https://user-images.githubusercontent.com/77106962/217357514-3cb41a90-3998-4323-8f4a-2084fd71cc70.png)

#### Kindly check the way you are creating a rental and the way you are accessing the `Person` and `Book` objects. 👍 😎

Reply:
fixed